### DISCUSSION TITLE
I don't understand why there is an `npmrc` option, but not a `yarnrc` option

### DISCUSSION THREAD

**Discussion by WoodyWoodsta** [renovatebot/renovate#21587] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

I'm trying to set up renovate on an org which has ~90 repos. So to maintain sanity, we keep a yarnrc in the home directory of our machines.

I thought that a mixture of `packageRules` and `hostRules` would allow us to keep this setup, and not have to commit the scopes into each repo (and future repos). `packageRules` don't seem to affect the `yarnrc.yml`, but the `hostRules` do - that in itself is head scratching point number 1.

But I'm most interested in why there is support for an `npmrc` option and not a `yarnrc` option? Is this a legacy thing? I saw that there used to be support, but it was removed in https://github.com/renovatebot/renovate/pull/10114.

It's my understanding that these two files exist for the same reasons, and behave in the same way, albeit with different structure. So it's odd that there would be a difference in config for each.

**Comment by rarkins** [comment]
We want to remove npmrc support eventually too. yarnrc was removed at a time when it contained no auth information.

I think relying on a yarnrc in home directories sounds like a code smell, if every tool and developer needs to manually set it up and ensure it's the same. What settings do you have in it which would impact Renovate's results? Eg scope->registry mappings?

**Reply by WoodyWoodsta** [reply]
> We want to remove npmrc support eventually too

I thought this might have been the case, and happy to hear the behaviour will become consistent.

"Code smell" is a stronger term than I would use, but ultimately I agree. The repos are all private packages, so the home directory config is relevant for internal development only, but this does go against various npm and yarn patterns. The settings are scope-registry mappings, yes.

I've already started moving the mappings into the repos anyway, so happy that the *rc configs will be removed entirely in the config.

Thank you for the help!!
